src/components/Display/Display.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Display from './Display';
+
+describe('Calculator', () => {
```

Comment:
```suggestion
describe('Display', () => {
```
I devour copy pasta myself 😋

Reply:
Also, I'll sometimes use:
```js
describe('<Display/>', () => {
```
Not that it is uber important, @kyleruschel do we have a convention on this?

Reply:
Thats nicer to me. Seems more meaningful.

Reply:
We don't have a convention for this particular instance. We have tests in the client now that follow your suggestion @jacobvenable and we have others that just name the component being tested inside a string. I think, if this is the top-level describe, following @jacobvenable's suggestion is a solid approach but I don't think you'd get dinged for this